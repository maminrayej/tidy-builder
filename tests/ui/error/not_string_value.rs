#[derive(tidy_builder::Builder)]
struct MyStruct {
    #[builder(each = 1)]
    args: Vec<String>,
}

fn main() {}
