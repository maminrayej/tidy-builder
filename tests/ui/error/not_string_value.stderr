error: Literal must be a string
 --> tests/ui/error/not_string_value.rs:3:22
  |
3 |     #[builder(each = 1)]
  |                      ^
